packages/mobile/src/screens/notifications-screen/NotificationsScreen.tsx

```diff
@@ -1,53 +1,38 @@
-import { memo, useCallback, useContext, useEffect } from 'react'
+import { useEffect } from 'react'
 
 import { useDrawerStatus } from '@react-navigation/drawer'
 import { markAllAsViewed } from 'audius-client/src/common/store/notifications/actions'
-import { StyleSheet, View } from 'react-native'
+import { View } from 'react-native'
 import { usePrevious } from 'react-use'
 
 import { useDispatchWeb } from 'app/hooks/useDispatchWeb'
-import { useTheme } from 'app/utils/theme'
+import { makeStyles } from 'app/styles'
 
-import { List } from './List'
-import { NotificationsDrawerNavigationContext } from './NotificationsDrawerNavigationContext'
-import TopBar from './TopBar'
+import { NotificationList } from './NotificationList'
+import { TopBar } from './TopBar'
 
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    position: 'absolute',
-    top: 0,
-    width: '100%',
-    height: '100%'
+const useStyles = makeStyles(({ palette }) => ({
+  root: {
+    backgroundColor: palette.background
   }
-})
+}))
 
-/**
- * A component that renders a user's notifications
- */
-export const NotificationsScreen = memo(() => {
+export const NotificationsScreen = () => {
```

Comment:
I did add the memo at some point to prevent the notifications screen from rerendering on bottom tab navigation. Can you double check to see if that is still happening?

Reply:
ahh good call, yeah will check

Reply:
re-added and added comment!